lib/mongodb.js

```diff
@@ -1870,30 +1873,71 @@ MongoDB.prototype.ping = function(cb) {
   }
 };
 
+// Determine if a property must have an ObjectID value
+function mustBeObjectID(propDef) {
+  if (!propDef) return false;
+  if (propDef.mongodb && ObjectIdTypeRegex.test(propDef.mongodb.dataType)) return true;
+  return false;
+}
+
+// Determine if strictObjectIDCoercion should be enabled
+function enableStrictObjectIDCoercion(modelCtor, options) {
```

Comment:
The name implies the function is going to enable coercion, which is not true AFAICT. Proposed name:

```suggestion
function isStrictObjectIDCoercionEnabled(modelCtor, options) {
```